rand_distr/Cargo.toml

```diff
@@ -30,6 +30,7 @@ alloc = ["rand/alloc"]
 std_math = ["num-traits/std"]
 
 serde = ["dep:serde", "rand/serde"]
+serde_with = ["dep:serde_with"]
```

Comment:
This doesn't show up either in API docs or the README. Could you add at least the latter please, preferably also a serialisation test?

Or we could just roll it under the `serde` feature. Either is acceptable.

Reply:
This was added in 0f5af6672 (PR #1292) by @Armavica 

Any chance they're still around and can comment?

Reply:
It was introduced for serializing const arrays, see the commit.

Reply:
Okay, so it should be just part of the `serde` feature and not a separate (previously implicit) `serde_with` feature.

I'll update this PR over the weekend for that.

Reply:
I agree!

Reply:
@waywardmonkeys a friendly reminder (if you'd be so kind)

Reply:
Done.